flink-docs/README.md

```diff
@@ -1,4 +1,4 @@
-<!--
+N<!--
```

Comment:
nit: why have this?

Reply:
thanks a lot for noticing this, this is not needed of course :)